lib/rover/data_frame.rb

```diff
@@ -430,13 +430,13 @@ def plot(x = nil, y = nil, type: nil)
           end
 
         scale = x_type == "temporal" ? {type: "utc"} : {}
-
```

Comment:
Please make sure there aren't unnecessary changes to lines.

Reply:
Must've been an accidental keyboard tap. Added the newline again.